sentry-core/src/main/java/io/sentry/core/Hub.java

```diff
@@ -77,14 +77,18 @@ public SentryId captureException(Throwable throwable) {
   public void close() {
     // Close the top-most client
     StackItem item = stack.peek();
-    item.client.close();
+    if (item != null) {
```

Comment:
Makes me wonder are we supposed to do when the caller does `bindClient(null)`.
Should we bind null? I'd say we bind `NoOpClient.Instance` and make sure this is never null

Reply:
the NPE check is for the `stack` and not for the `client` itself, but agreed with `NoOpClient.Instance`

Reply:
Right, I meant we're missing one for `client` here.

Reply:
`client` can't be null either, its created in the constructor or via `bindClient` but then we have `NoOp`